docs/DeveloperGuide.adoc

```diff
@@ -315,33 +324,333 @@ _{More to be added}_
 
 (For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
 
+
 [discrete]
-=== Use case: Delete person
+=== Use case: Add a team member
 
-*MSS*
+*Main Success Scenario*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1. User requests to add a team member
+2. User specifies the name of the team member
```

Comment:
This seems to be that at step 1, the user types in the command, then +Work waits for the user to type in the name of the team member. May I suggest adding a step in between where `+Work notifies user to key in the name of team member`

Reply:
This applies for your other use cases too

Reply:
Also, before use case ends, it may be good to notify the user that their action was successful.